### ISSUE TITLE
Dump in ZCL_EXCEL_READER_2007->load_worksheet  CONVT_NO_NUMBER

### ISSUE THREAD

**Issue by rotda** [abap2xlsx/abap2xlsx#550]
Hi ,

another issue with my very complex excel file in the reader class
It seems sometime excel have an span property of row with two values.
And then it dumps with message 

 "240 5454:5" cannot be interpreted as a number

Workaround , but not the best one because you lose the second value. But it works with my excel without any issues.
 
Line : 2427 of ZCL_EXCEL_READER_2007

```
    FIND ALL occurrences OF ':' IN ls_row-spans MATCH COUNT lv_counter.
    if lv_counter = 1.
        SPLIT ls_row-spans AT ':' INTO lv_min_col_s lv_max_col_s.
    else.
*    BugFix Sometimes excel have two values
* Work Around use the first one
      data lv_row_span_1 type string.
      data lv_row_span_2 type string.
      SPLIT ls_row-spans AT ' ' INTO lv_row_span_1 lv_row_span_2.
      SPLIT lv_row_span_1  AT ':' INTO lv_min_col_s lv_max_col_s.
    ENDIF.
```

**Comment by sandraros** [comment]
Sorry for the late answer. I can't reproduce your issue. Could you provide an Excel file to reproduce it please?
Thank you.

**Comment by sandraros** [comment]
I think I have just reproduced the issue and I proposed a temporary fix in this other same issue: #407. If you agree, we may close your issue and continue in the other one. Thank you.

**Comment by sandraros** [comment]
No answer. Closing.
